src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,24 @@
+package codeu.chat.common;
+
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  private Uuid parseResult;
+
+  public ServerInfo() {
+    try {
+      parseResult = Uuid.parse(SERVER_VERSION);
+    } catch (IOException exception) {
+      exception.getMessage();
```

Comment:
I tried doing that before, but an error would occur saying that there was an unchecked IOException. I was talking to other CodeUers, and they said they also had this problem. One of them gave me this solution, and my code finally compiled after I used it.